framework/build/enyo-build.js

```diff
@@ -5872,8 +5866,7 @@ return enyo.fetchConfigFile("$enyo/../framework_config.json");
 }, enyo.fetchDeviceInfo = function() {
 return window.PalmSystem ? JSON.parse(PalmSystem.deviceInfo) : undefined;
 }, enyo.requiresWindow(function() {
-if ((window.PalmSystem) && (window.PalmSystem.getResource) && (!window.palmGetResource)) {window.palmGetResource = PalmSystem.getResource;}
-window.addEventListener("load", enyo.ready, !1), window.addEventListener("resize", enyo.sendOrientationChange, !1), Mojo = window.Mojo || {}, Mojo.lowMemoryNotification = function(a) {
+window.addEventListener("load", enyo.ready, !1), window.addEventListener("resize", enyo.sendOrientationChange, !1), Mojo = window.Mojo || {}, window.PalmSystem && window.PalmSystem.getResource && !window.palmGetResource && (window.palmGetResource = PalmSystem.getResource), Mojo.lowMemoryNotification = function(a) {
```

Comment:
This looks a bit off but might be wrong

Reply:
For this one, I think it's just that the lines were not in the same order in the minified version and in the source code. But looking at the code, the behavior should be correct anyway.